src/app/campaign/list/campaign-list-paging.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+
+@Component({
+  selector: 'grove-campaign-list-paging',
+  template: `
+    <div class="paging-container">
+      <button class="pager"
+        [disabled]="isPageActive(1)"
+        (click)="!isPageActive(1) && showPage.emit(1)"
+        title="Page 1">|«</button>
+      <button class="pager"
+        [disabled]="isPageActive(1)"
+        (click)="!isPageActive(1) && showPage.emit(currentPage - 1)"
+        title="Page {{ currentPage > 1 ? currentPage - 1 : 1 }}">«</button>
+
+      <button
+        *ngFor="let x of ' '.repeat(totalPages).split(''), let i = index"
```

Comment:
Array.from is not valid in the template. Not having a simple index without an iterator in templates can be annoying sometimes. I think this code will clean itself up once we finish implementing paging with #21 because it won't be displaying all the pages.